recipes/fenics-basix/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "fenics-basix" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  # 0.4.1 isn't on PyPI
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/fenics/basix/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 34de61ca3e39b1aa0392ec6dad292bb9ffaf965c0734a8638427b9d850180dc7
+
+build:
+  skip: true  # [win]
+  number: 0
+
+outputs:
+  - name: fenics-libbasix
+    script: build-libbasix.sh
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+        - make
+        - pkg-config
+      host:
+        - libblas
```

Comment:
```suggestion
```
The [link inspector says](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=505698&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1941) that libblas is not used.